### PR TITLE
Added support for: individual files, names with spaces, relative paths and symlinks

### PR DESCRIPTION
**Pull request by nmaludy**
This pull request contains several new features:

- Support for individual files - Some of the import functions in the shell script only supported importing whole directories, even though the Ruby code supported importing individual files. I've modified the shell script to support both scenarios

- Support for names with spaces - Lots of changes here. The shell scripts were not very forgiving for importing/exporting file paths that contained spaces. I added quotes in the the shell scripts to support this.  I also changed the export filename regex to replace ' ' with '_' so that by default the filenames do not have spaces in them.

- Support for relative paths and symlinks - Previously, running miqimport/miqexport on a relative path would fail with some odd errors. All that was needed here was to add a "readlink" on the path argument to resolve the absolute path from the relative.

Please let me know if you have suggestions or find any bugs.

Thanks,
Nick

### CODE REVIEW

**Comment by branic** [comment]
@nmaludy, I like this PR, but I'm concerned about breaking exports that have previously been done where the filenames have spaces in them. Changing spaces to _ would have a side affect on files that have been commited to a git repo.

I would like to see replacing the space be an option that can be passed in or a switch to maintain old behavior along with relevant comments added to the README file.

Thoughts?

- Brant

**Comment by nmaludy** [comment]
@branic I've update the PR with a command line option to preserve the old behavior of spaces in filenames (-s or --keep-spaces). I've also moved the regexes for the replacement into a common file called rhconsulting_illegal_chars.rb so that if/when we need to modify this in the future it's in a common spot.

Other comments are welcome.

**Comment by nmaludy** [comment]
I saw that @ewannema added several options for importing domains that we needed. I've added command line switches to expose these options in the miqimport bash script. Also made overwrite=true work for CloudForms 4.0.

File: bin/miqexport [hunk 1]
```diff
@@ -1,185 +1,157 @@
...
-    echo "Export directory doesn't exist!"
-    exit 1
-  fi
+  assert_num_args $# 1
+  EXPORT_DIR=$(readlink -v -f "$1")
+  assert_export_dir_exists "$EXPORT_DIR"
 
   pushd /var/www/miq/vmdb
-  bin/rake "rhconsulting:provision_dialogs:export[${EXPORT_DIR}]"
+  bin/rake --trace "rhconsulting:provision_dialogs:export[${EXPORT_DIR},keep_spaces=${KEEP_SPACES}]"
```
Comment (branic) [inline bin/miqexport]: @nmaludy is the --trace really needed? If not please remove it.

File: cfme-rhconsulting-scripts.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:      cfme-rhconsulting-scripts
-Version:   0.8
-Release:   2
+Version:   0.9
```
Comment (branic) [inline cfme-rhconsulting-scripts.spec]: The /var/www/miq/vmdb/lib/tasks/rhconsulting_illegal_chars.rb and /var/www/miq/vmdb/lib/tasks/rhconsulting_options.rb files need to be added to the %files section so they get installed.

Also, the %changelog section should be updated.

**Comment by nmaludy** [comment]
@branic Made all requested changed.

**Pull request merged**